MarkBind/markbind
# Change seamless panels to inherit any parent's colours

**What is the purpose of this pull request?**




**Overview of changes:**
#1697 
Changed the seamless panels so that the background is transparent and it inherits the colours of any parents its contained in. 

**Anything you'd like to highlight/discuss:**
As pointed out in the issue, this might break some edge cases (e.g. a seamless panel in a `<box background-color="black" color="black">` box). 

**Testing instructions:**
I've added some examples in the userguide to show the new look. `markbind serve -d` the docs to see the changes and try out different cases to see if it works or if there's any improvements.



**Proposed commit message: (wrap lines at 72 characters)**
Change seamless panels to inherit colours of any parents



---

**Checklist:** :ballot_box_with_check:






---

**Reviewer checklist:**

Indicate the [SEMVER](https://semver.org/) impact of the PR:

At the end of the review, please label the PR with the appropriate label: `r.Major`, `r.Minor`, `r.Patch`.

Breaking change release note preparation (if applicable):
- To be included in the release note for any feature that is made obsolete/breaking

> Give a brief explanation note about:
>   - what was the old feature that was made obsolete
>   - any replacement feature (if any), and
>   - how the author should modify his website to migrate from the old feature to the replacement feature (if possible).
